.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+### Description
+[Describe what this change achieves]
+ 
+### Issues Resolved
+[List any issues this PR will resolve]
+ 
+### Check List
+- [ ] New functionality includes testing.
+  - [ ] All tests pass
```

Comment:
I'm actually feeling like we should change the core repo's check boxes because sometimes it won't be new functionality so the first box is not checked off usually but the rest are. Maybe these can be two no-indent checkboxes?

Reply:
It makes sense. I also reordered them a bit.